### PR TITLE
New Flask cheat sheet.

### PR DESCRIPTION
**Pull request by pjhampton**
#### Description
Flask is a Python microframework.

#### Data Source
http://flask.pocoo.org/

#### Example Queries
flask cheat sheet

#### Triggers
No triggers applied.

**Screenshot 1**

screen shot 2016-03-26 at 20 56 35

**Screenshot 2**

screen shot 2016-03-26 at 20 56 11

https://duck.co/ia/view/flask_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mohan08p** [comment]
@pjhampton : kindly change the template type to "code"

**Comment by pjhampton** [comment]
@mohan08p

Hey dude, 

Look closer. This is just the duckpan auto generated template? Thanks for trying to help though.

**Comment by mohan08p** [comment]
@pjhampton : ok. cool. yeah then `terminal` is better. :+1:

**Comment by pjhampton** [comment]
@mohan08p I will have it up in the next couple of hours. I will ping you when it is ready if you would be up for reviewing it for me? :smile:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=flask cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/flask_cheat_sheet_mobile.png)

**Comment by mohan08p** [comment]
looks good. :+1:

File: share/goodie/cheat_sheets/json/flask.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            {
+                "key": "if __name__ == \"__main__\":\n\t\tapp.run()" ,
+                "val": "If involked directly, execute"
+            }
+        ],
+
+        "Creating Pages": [
+            {
+                "key": "@app.route('/')\ndef index():\n\t\treturn 'hello, world'",
+                "val": "Renders the text 'hello, world in the browser"
```
Comment (mohan08p) [inline share/goodie/cheat_sheets/json/flask.json]: oh.. just remove the single exclamatory or complete it. 'hello, world'

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/flask.json]: Well spotted man

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=flask cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/flask_cheat_sheet_mobile.png)

**Comment by anselal** [comment]
You have a typo in the return statement at "Minimum Viable Example"

**Comment by pjhampton** [comment]
@anselal many thanks

**Comment by anselal** [comment]
How can I be added as a contributor to this cheat sheet ?

**Comment by pjhampton** [comment]
@anselal I believe you need to add a completely separate PR once the cheat sheet is in production - which this one isn't. I may be wrong though ---> thoughts @duckduckgo/duckduckhack-mentors?

**Comment by anselal** [comment]
@pjhampton You can edit the Contributors at https://duck.co/ia/view/flask_cheat_sheet

**Comment by pjhampton** [comment]
@anselal Yep. Credit where credit is due. I've added you. 

Thanks for your help :+1: I would avoid making any more contributions to this until @moollaza has merged this.

**Comment by anselal** [comment]
@pjhampton You can give write access to your code by opening settings on your project and add any users you want under collaborators.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=flask cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/flask_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
This LGTM :+1:

Thanks again for taking the time to contribute! We really appreciate it :smiley:

**Comment by daxtheduck** [comment]
@pjhampton this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**